src/ui/molecules/card/index.css

```diff
@@ -13,3 +13,25 @@
 .card h2 {
   margin: 0;
 }
+
+.card .github:before {
+  background: url("../../../assets/github-mark.svg");    
+  content: '';
+  display: inline-block;
+  background-size: 20px 20px;
+  width: 20px;
+  height: 20px;
+  vertical-align: text-bottom;
+  margin-right: 5px;  
+ }
+
+.card .gitlab:before {
```

Comment:
One last small thing before we can merge. While browsers still support the single `:` syntax, the preferred syntax for pseudo-elements is `::`. You can read more on MDN Web Docs - https://developer.mozilla.org/en-US/docs/Web/CSS/::before#syntax

Reply:
That explains why I've seen both before. I'll use `::` from now on. Thanks!